engo_mobile.go

```diff
@@ -0,0 +1,223 @@
+//+build android
+
+package engo
+
+import (
+	"engo.io/gl"
+	"fmt"
+	"image"
+	"image/draw"
+	_ "image/png"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/golang/freetype"
+	"github.com/golang/freetype/truetype"
+	"golang.org/x/mobile/app"
+	"golang.org/x/mobile/asset"
+	"golang.org/x/mobile/event/lifecycle"
+	"golang.org/x/mobile/event/paint"
+	"golang.org/x/mobile/event/size"
+	"golang.org/x/mobile/event/touch"
+	"golang.org/x/mobile/exp/app/debug"
+	"golang.org/x/mobile/exp/gl/glutil"
+	mobilegl "golang.org/x/mobile/gl"
+)
+
+var (
+	Gl *gl.Context
+	sz size.Event
+
+	gameWidth, gameHeight     float32
+	windowWidth, windowHeight float32
+
+	msaaPreference int
+)
+
+func CreateWindow(title string, width, height int, fullscreen bool, msaa int) {
```

Comment:
Although we said try to document any new things you add, I'll let this one slide and let it get picked up by the upcoming documentation PRs.